cli.py

```diff
@@ -0,0 +1,166 @@
+import argparse
+from sys import exit
+from zokrates.gadgets.pedersenHasher import PedersenHasher
+from zokrates.babyjubjub import Point
+from zokrates.eddsa import PrivateKey, PublicKey
+from zokrates.field import FQ
+
+
+def main():
+    parser = argparse.ArgumentParser(description="pycrypto command-line interface")
+    subparsers = parser.add_subparsers(dest="subparser_name")
+
+    # pedersen hash subcommand
+    pedersen_parser = subparsers.add_parser(
+        "hash",
+        help="Compute a 256bit Pedersen hash. Preimage size is set to 512bit as default",
+    )
+    pedersen_parser.add_argument(
+        "preimage", nargs=1, help="Provide preimage as hexstring"
+    )
+    pedersen_parser.add_argument(
+        "-s", "--size", type=int, help="Define message size in bits", default=64
+    )
+    pedersen_parser.add_argument(
+        "-p", "--personalisation", help="Provide personalisation string", default="test"
+    )
+
+    # batch pedersen hash subcommand
+    pedersen_hasher_parser = subparsers.add_parser(
+        "batch_hasher",
+        help="Efficiently compute multiple Pedersen hashes. Support for stdin and alive promt",
+    )
+
+    pedersen_hasher_parser.add_argument(
+        "-s", "--size", type=int, help="Define message size in bits", default=64
+    )
+    pedersen_hasher_parser.add_argument(
+        "-p", "--personalisation", help="Provide personalisation string", default="test"
+    )
+
+    # keygen subcommand
+    keygen_parser = subparsers.add_parser(
+        "keygen",
+        help="Returns space separated hex-string for a random private/public keypair on BabyJubJub curve",
+    )
+    keygen_parser.add_argument(
+        "-p",
+        "--from_private",
+        help="Provide existing private key as hexstring (64 chars)",
+    )
+
+    # eddsa signature generation subcommand
+    sig_gen_parser = subparsers.add_parser(
+        "sig-gen",
+        help="Returns eddsa signature as space separated hex-string. Private key and message needs to be provided",
+    )
+    sig_gen_parser.add_argument(
+        "private_key", nargs=1, help="Provide public key as hexstring (64chars)"
```

Comment:
This should be "private key" instead of "public key"

Reply:
good catch.